### PR TITLE
[KYUUBI #1675][FEATURE] Redact secret information from SparkSQLEngine log

### PR DESCRIPTION
**Pull request by minyk**
### _Why are the changes needed?_

Secret informations on log is dangerous on the production environment. Secret info should be redacted in some way.

### _How was this patch tested?_
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [x] Add screenshots for manual tests if appropriate
capture from Spark Driver's STDERR
Screen Shot 2022-01-04 at 11 56 43 AM

- [x] [Run test](https://kyuubi.readthedocs.io/en/latest/develop_tools/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

**Review by ulysses-you — APPROVED** [review]

File: kyuubi-common/src/main/scala/org/apache/kyuubi/operation/AbstractOperation.scala [hunk 1]
```diff
@@ -80,7 +82,7 @@ abstract class AbstractOperation(opType: OperationType, session: Session)
       case _ =>
     }
     info(s"Processing ${session.user}'s query[$statementId]: ${state.name} -> ${newState.name}," +
-      s" statement: $statement$timeCost")
+      s" statement: $redactedStatement$timeCost")
```
Comment (ulysses-you) [inline kyuubi-common/src/main/scala/org/apache/kyuubi/operation/AbstractOperation.scala:85]: not related this PR, shall we and `, ` between `$redactedStatement` and `$timeCost`  ?

**Review by yaooqinn — APPROVED** [review]
LGTM

**Comment by yaooqinn** [comment]
thanks, merged to master

File: kyuubi-common/src/main/scala/org/apache/kyuubi/operation/AbstractOperation.scala (hunk 1 above)
Comment (minyk) [reply kyuubi-common/src/main/scala/org/apache/kyuubi/operation/AbstractOperation.scala:85]: @ulysses-you comma is already added from here: https://github.com/apache/incubator-kyuubi/blob/e4f5ba7010dd497f40d51f5677b16d3cf7b882d4/kyuubi-common/src/main/scala/org/apache/kyuubi/operation/AbstractOperation.scala#L81

Comment (ulysses-you) [reply kyuubi-common/src/main/scala/org/apache/kyuubi/operation/AbstractOperation.scala:85]: ah I see, thank you !

**Pull request closed without merging**